diff --git a/nexus/src/main/kotlin/tech/libeufin/nexus/server/NexusServer.kt b/nexus/src/main/kotlin/tech/libeufin/nexus/server/NexusServer.kt
index 4f1abc81..099baaff 100644
--- a/nexus/src/main/kotlin/tech/libeufin/nexus/server/NexusServer.kt
+++ b/nexus/src/main/kotlin/tech/libeufin/nexus/server/NexusServer.kt
@@ -1057,11 +1057,7 @@ fun serverMain(port: Int) {
         environment = applicationEngineEnvironment {
             connector {
                 this.port = port
-                this.host = "127.0.0.1"
-            }
-            connector {
-                this.port = port
-                this.host = "[::1]"
+                this.host = "0.0.0.0"
             }
             module(nexusApp)
         }
diff --git a/sandbox/src/main/kotlin/tech/libeufin/sandbox/Main.kt b/sandbox/src/main/kotlin/tech/libeufin/sandbox/Main.kt
index 16a3ccdc..eb428f97 100644
--- a/sandbox/src/main/kotlin/tech/libeufin/sandbox/Main.kt
+++ b/sandbox/src/main/kotlin/tech/libeufin/sandbox/Main.kt
@@ -1606,11 +1606,7 @@ fun serverMain(port: Int) {
         environment = applicationEngineEnvironment{
             connector {
                 this.port = port
-                this.host = "127.0.0.1"
-            }
-            connector {
-                this.port = port
-                this.host = "[::1]"
+                this.host = "0.0.0.0"
             }
             module(sandboxApp)
         },
